object/poster.go

```diff
@@ -0,0 +1,44 @@
+package object
+
+type Poster struct {
+	Name         string `xorm:"varchar(50) notnull pk" json:"name"`
+	Advertiser   string `xorm:"varchar(40)" json:"advertiser"`
+	Link         string `xorm:"varchar(500)" json:"link"`
+	Picture_link string `xorm:"varchar(500)" json:"picture_link"`
+}
+
+func AddPoster(poster Poster) bool {
+	affected, err := adapter.engine.Insert(poster)
+	if err != nil {
+		panic(err)
+	}
+
+	return affected != 0
+}
+
+func GetPoster(name string) *Poster {
+	poster := Poster{Name: name}
+	existed, err := adapter.engine.Get(&poster)
+	if err != nil {
+		panic(err)
+	}
+
+	if existed {
+		return &poster
+	} else {
+		return nil
+	}
+}
+
+func UpdatePoster(name string, poster Poster) bool {
```

Comment:
If the struct changed, `UpdatePoster` should be changed as well, update `deleted` filed.